src/components/BodyTextListItem/BodyTextListItem.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { useFocus } from '../../hooks';
+import { useFocus } from '../../hooks/useFocus';
 import colors from '../../theme/colors.scss';
 
 import './BodyTextListItem.scss';
 
 const prefixCls = 'kai-btl';
 
-const BodyTextListItem = React.memo(
+interface LocalProps {
+  header: string;
+  body?: string;
+  focusColor?:string;
+  forwardedRef?:any;
```

Comment:
Is there no equivalent to the propType I used

Reply:
Not that I could find, no. If you find one, happy to replace it.